src/serlio/utils/MayaUtilities.h

```diff
@@ -28,4 +28,21 @@ void forAllAttributes(const MFnDependencyNode& node, F func) {
 	}
 }
 
+// credits: https://www.fluentcpp.com/2016/12/08/strong-types-for-strong-interfaces/
+template <typename T, typename Parameter>
+class NamedType {
+public:
+	explicit NamedType(T const& value) : value_(value) {}
+	explicit NamedType(T&& value) : value_(std::move(value)) {}
+	T& get() {
+		return value_;
+	}
+	T const& get() const noexcept {
+		return value_;
+	}
+
```

Comment:
we might want to make the default dtor, copy/move ctors and assigment operators protected to discourage direct use of `NamedType`

Reply:
i leave it for the moment, it is so short and simple :-)